packages/cli/src/commands/epochs/finish.ts

```diff
@@ -0,0 +1,36 @@
+import chalk from 'chalk'
+import { BaseCommand } from '../../base'
+import { displaySendTx } from '../../utils/cli'
+import { CustomFlags } from '../../utils/command'
+
+export default class Finish extends BaseCommand {
+  static description = 'Finishes next epoch process.'
+
+  static flags = {
+    ...BaseCommand.flags,
+    from: CustomFlags.address({ required: true }),
```

Comment:
why is this required? can it not default to the connected account?

Reply:
this is copy&paste from switch command - @shazarre should this be required ?

Reply:
sorry ignore me. this is typical